include/utilities/realtime/dataman/DataManager.h

```diff
@@ -0,0 +1,44 @@
+#ifndef DATAMANAGER_H_
+#define DATAMANAGER_H_
+
+#include "utilities/realtime/dataman/DataMan.h"
+
+namespace adios
+{
+namespace realtime
+{
+
+class DataManager : public DataMan
+{
+public:
+    DataManager();
```

Comment:
Since both constructors and destructor have empty implementations, they should use `default` instead.  Also, always make the destructor of an inherited class virtual; i.e:
```
DataManager() = default;
virtual ~DataManager() = default;
```